Description: Use canned version of ThreadedGB example that fails on mips64el
Author: Doug Torrance <dtorrance@piedmont.edu>
Bug: https://github.com/Macaulay2/M2/issues/1463
Last-Update: 2020-12-06

--- a/M2/Macaulay2/packages/ThreadedGB.m2
+++ b/M2/Macaulay2/packages/ThreadedGB.m2
@@ -349,10 +349,46 @@
       The interesting part of the output may be the actual  lineages of the basis polynomials,
       in addition to the Groebner basis itself.
       Here is a verbose example when the Groebner basis is trivial.
-    Example
-      QQ[a..d]
-      I=ideal( -c^3+a^2+b*d, a*b*c-1,a*b*c)
-      T = tgb(I,2,Verbose=>true)
+    CannedExample
+      i10 : QQ[a..d]
+
+      o10 = QQ[a, b, c, d]
+
+      o10 : PolynomialRing
+
+      i11 : I=ideal( -c^3+a^2+b*d, a*b*c-1,a*b*c)
+
+                      3    2
+      o11 = ideal (- c  + a  + b*d, a*b*c - 1, a*b*c)
+
+      o11 : Ideal of QQ[a, b, c, d]
+
+      i12 : T = tgb(I,2,Verbose=>true)
+      You turned on Verbose! You will be notified of each new S-polynomial task created and each new GB element added to the HashTable as we go.
+      Scheduling a task for lineage (0-1)
+      Scheduling a task for lineage (0-2)
+      Scheduling a task for lineage (1-2)
+      Scheduling task for lineage ((0-1)-0)
+      Scheduling task for lineage ((0-1)-1)
+      Scheduling task for lineage ((0-1)-2)
+      Adding the following remainder to GB: -1 from lineage (1-2)
+      Adding the following remainder to GB: -a^3*b-a*b^2*d+c^2 from lineage (0-1)
+      Scheduling task for lineage ((0-2)-0)
+      Scheduling task for lineage ((0-2)-1)
+      Scheduling task for lineage ((0-2)-2)
+      Scheduling task for lineage ((0-2)-(1-2))
+      Scheduling task for lineage ((0-2)-(0-1))
+      Adding the following remainder to GB: -a^3*b-a*b^2*d from lineage (0-2)
+      Found 1 or -1 in the Groebner basis; reducing now.
+
+      o12 = HashTable{(0-1) => null}
+                      (0-2) => null
+                      (1-2) => 1
+                      0 => null
+                      1 => null
+                      2 => null
+
+      o12 : HashTable
     Text
       In particular, the lineages of null values tell us what S-polynomials didn't reduce to zero until $1$ was found as
       a remainder.
